ui/src/pages/AllStories.tsx

```diff
@@ -0,0 +1,288 @@
+import Paper from "@material-ui/core/Paper";
+import {
+  createStyles,
+  Theme,
+  WithStyles,
+  withStyles,
+} from "@material-ui/core/styles";
+import Switch from "@material-ui/core/Switch";
+import TableCell from "@material-ui/core/TableCell";
+import clsx from "clsx";
+import * as React from "react";
+import {
+  AutoSizer,
+  Column,
+  Table,
+  TableCellRenderer,
+  TableHeaderProps,
+} from "react-virtualized";
+import useSWR from "swr";
+
+import { colors } from "../styles/colors";
+import { Story } from "../types";
+
+declare module "@material-ui/core/styles/withStyles" {
+  interface BaseCSSProperties {
+    flip?: boolean;
+  }
+}
+
+const styles = (theme: Theme) =>
+  createStyles({
+    flexContainer: {
+      display: "flex",
+      alignItems: "left",
+      boxSizing: "border-box",
+    },
+    table: {
+      "& .ReactVirtualized__Table__headerRow": {
+        flip: false,
+        paddingRight: theme.direction === "rtl" ? "0 !important" : undefined,
+        backgroundColor: colors.primaryLight4,
+        fontWeight: 800,
+      },
+      "& .ReactVirtualized__Grid ReactVirtualized__Table__Grid": {},
+    },
+    tableRow: {
+      cursor: "pointer",
+      color: colors.grey,
+    },
+    tableRowHover: {
+      "&:hover": {
+        backgroundColor: colors.black,
+      },
+    },
+    tableCell: {
+      flex: 2,
+    },
+    noClick: {
+      cursor: "initial",
+    },
+  });
+
+interface ColumnData {
+  dataKey: string;
+  label: string;
+  numeric?: boolean;
```

Comment:
we shouldn't mix undefined and boolean types. Let's enforce this to be a boolean and write a data transform if needed.